src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +622,106 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
+    ) -> bool:  # TODO: add the anomaly type once that's added to Sentry
```

Comment:
you can leave the `TODO:` in the docstring i guess?
Or maybe just a comment below the docstring